Hazel/src/Platform/OpenGL/OpenGLBuffer.h

```diff
@@ -15,6 +15,8 @@ namespace Hazel {
 
 		virtual const BufferLayout& GetLayout() const override { return m_Layout; }
 		virtual void SetLayout(const BufferLayout& layout) override { m_Layout = layout; }
+
+		friend class OpenGLVertexArray;
```

Comment:
Why is `friend class OpenGLVertexArray;` declared twice?

Reply:
I think it is once for each of the two classes in this file.

Reply:
This is indeed once for the `OpenGLVertexBuffer` and once for the `OpenGLIndexBuffer`